jekyll/_api/v1-reference.md

```diff
@@ -278,7 +275,7 @@ You can retry a build with ssh by swapping "retry" with "ssh":
 
 {{ site.data.api.cancel_build | api_endpoint }}
 
-<h2 id="new-build">Trigger a new Build</h2>
+<h2 id="new-build">Trigger a new Build by Number</h2>
```

Comment:
Not sure if "by Number" makes sense -- suggest changing this to "Trigger a new Job", as that's what this endpoint does right now.

Reply:
thanks, fixed